app/connectors/propertyLinking/PropertyLinkConnector.scala

```diff
@@ -177,6 +177,15 @@ class PropertyLinkConnector @Inject()(config: ServicesConfig, http: HttpClient)(
   def getMyOrganisationAgents()(implicit hc: HeaderCarrier): Future[AgentList] =
     http.GET[AgentList](s"$baseUrl/owner/agents")
 
+  def sendAgentRelationshipRequest(agentRelationshipRequest: AgentRelationshipRequest)(
+        implicit hc: HeaderCarrier): Future[Unit] = {
+    http
+      .POST[AgentRelationshipRequest, HttpResponse](
+        s"$baseUrl/my-organisation/appointmentChanges",
+        agentRelationshipRequest)
+      .map(_ => ()) //fixme is it async ?
```

Comment:
this is async. just capture the response (which will be a generic success response) and pass this up.

Reply:
addressed

Reply:
I also dont know if we should be expanding there pattern all the way to the frontend. if we had several endpoints in voa-property-linking backend that do appoint and revoke seperately it might make us less coupled to modernised behaviour.